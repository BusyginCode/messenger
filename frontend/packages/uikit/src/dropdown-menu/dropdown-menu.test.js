import React from 'react'
import { shallow } from 'enzyme'
import DropdownMenu from './dropdown-menu'

describe('uikit/DropdownMenu', () => {
  it('should render correctly with custom trigger', () => {
    const wrapper = shallow(
      <DropdownMenu
        placement="topLeft"
        options={[{ id: 1, component: 'div' }, { id: 2, component: 'div' }]}
        trigger={['click']}
      >
        <button>...</button>
      </DropdownMenu>
    )
    expect(wrapper).toMatchSnapshot()
  })
})
