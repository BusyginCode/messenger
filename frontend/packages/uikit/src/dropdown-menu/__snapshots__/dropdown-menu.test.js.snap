// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/DropdownMenu should render correctly with custom trigger 1`] = `
<Dropdown
  onVisibleChange={[Function]}
  options={
    Array [
      Object {
        "component": [Function],
        "id": 1,
      },
      Object {
        "component": [Function],
        "id": 2,
      },
    ]
  }
  overlayClassName="overlay topLeft"
  placement="topLeft"
  trigger={
    Array [
      "click",
    ]
  }
  visible={false}
>
  <button>
    ...
  </button>
</Dropdown>
`;
